srcpkgs/KataGo-OpenCL/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'KataGo-OpenCL'
+pkgname=KataGo-OpenCL
+version=1.11.0
+revision=1
+wrksrc="KataGo-${version}"
+build_wrksrc=cpp
+build_style=cmake
+configure_args="-DUSE_BACKEND=OPENCL -DNO_GIT_REVISION=1"
+makedepends="ocl-icd-devel zlib-devel libzip-devel"
+depends="mesa-opencl"
```

Comment:
People are expected to install right implementation themself.

Reply:
> People are expected to install right implementation themself.

of OpenCL? How would that change this section? Do I just remove that depends= line?

Reply:
yes